test/Chirp.InfrastructureTests/RepositoryTests/AuthorRepositoryTest.cs

```diff
@@ -14,480 +15,390 @@ public class AuthorRepositoryTest
         };
-
-        // Act
-        int initialAuthorCount = context.Users.Count();
         
-        authorRepository.AddAuthor(authorDto2);
-
-        context.SaveChanges();
+        _authorRepository.AddAuthor(author4);
 
         int updatedAuthorCount = context.Users.Count();
 
         // Assert
         Assert.Equal(initialAuthorCount + 1, updatedAuthorCount);
     }
-
+    
+    
+    // ----- Get Author Methods ----- //
     [Fact]
     public void GetAuthorById_ShouldReturnCorrectAuthor()
     {
-        // Arrange
-        var authorRepository = new AuthorRepository(context);
-        
-        // Created to test for the correct author
-        Author theAuthor = null;
-        
-        // Create 3 authors
-        for (int i = 0; i < 3; i++)
-        {
-            Author authorDto = new Author
-            {
-                Id = Guid.NewGuid(),
-                UserName = "TestAuthor" + i,
-                Email = "mock" + i + "@email.com"
-            };
-            
-            // Set theAuthor to the middle author
-            if (i == 1)
-            {
-                theAuthor = authorDto;
-            }
-
-            context.Users.Add(authorDto);
-        }
-
-        context.SaveChanges();
-
         //Act
-        Author expectedAuthor = theAuthor;
-        Author returnedAuthor = authorRepository.GetAuthorById(theAuthor.Id);
+        Author expectedAuthor = _author2;
+        Author returnedAuthor = _authorRepository.GetAuthorById(_author2.Id);
 
         //Assert
         Assert.Equal(expectedAuthor, returnedAuthor);
     }
-
+    
     [Fact]
-    public void GetAuthorByEmail_ShouldReturnCorrectAuthor()
+    public void GetAuthorByIdAsync_ShouldReturnCorrectAuthor()
     {
-        // Arrange
-        var authorRepository = new AuthorRepository(context);
-        
-        // Created to test for the correct author
-        Author theAuthor = null;
-        
-        // Create 3 authors
-        for (int i = 0; i < 3; i++)
-        {
-            Author authorDto = new Author
-            {
-                Id = Guid.NewGuid(),
-                UserName = "TestAuthor" + i,
-                Email = "mock" + i + "@email.com"
-            };
-            
-            // Set theAuthor to the middle author
-            if (i == 1)
-            {
-                theAuthor = authorDto;
-            }
-
-            context.Users.Add(authorDto);
-        }
-
-        context.SaveChanges();
-
         //Act
-        Author expectedAuthor = theAuthor;
-        Author returnedAuthor = authorRepository.GetAuthorByEmail(theAuthor.Email);
+        Author expectedAuthor = _author2;
+        Author? returnedAuthor = _authorRepository.GetAuthorByIdAsync(_author2.Id).Result;
 
         //Assert
         Assert.Equal(expectedAuthor, returnedAuthor);
     }
     
     [Fact]
-    public void GetAuthorByIdAsync_ShouldReturnCorrectAuthor()
+    public void GetAuthorByName_ShouldReturnCorrectAuthorDTO()
```

Comment:
Why does this one method's name end in "DTO" when none of the other 'ShouldReturnCorrectAuthor" test does not?

Simply a naming thing. Please choose one or the other.